docs/source/pillars/data_management.md

```diff
@@ -149,6 +149,24 @@ The ability of the TRE operator to ensure outputs are safely published and share
     Results will be produced which address the project's purpose, some of which may be outputs that are removed from the TRE.
     Understanding what these outputs are likely to be and their sensitivity as early as possible will help prepare for their processing and publication.
   - Recommended
+* - 3.3.3.
+  - You must have a documented process for disclosure control of output files from the TRE.
+  - This process should define expected risks and how to mitigate them.
+    All TRE outputs must be subject to this process.
+    You might choose to follow existing guidelines, for example around statistical disclosure.
+  - Mandatory
+* - 3.3.4.
+  - You must have a process for assigning responsibility for output checking.
+  - A named person or team should be given responsibility for checking outputs.
+    This person or team must follow your disclosure control process and will be responsible for any automated parts of this process.
```

Comment:
What is important here. Is it that we know who does the output checking or that there are people able to do it?

I think it would be better to introduce an output checker role and refer to that here.

Reply:
We still haven't filled out roles: https://satre-specification.readthedocs.io/en/latest/roles.html

Perhaps now is a good time to have a look at this? Happy to have a first go

Reply:
Talk to @machintim also as he's done a bit of work here.

Reply:
To avoid blocking PRs could we either:
- remove this for now
- or keep it, and replace it with a link to a role in a follow-up?

Reply:
Or just link to a role called `Output Checker` that doesn't have any content yet?